backend/agents/persona_agent.py

```
@@ -462,10 +543,10 @@ async def chat(self,
                         "message_type": "user",
                         "user_progress_id": str(user_progress_id),
                         "scene_id": str(scene_id),
-                        "timestamp": str(datetime.now())
+                        "timestamp": str(datetime.now()),
```

Ruff (0.14.1)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)